### PR TITLE
ZCS-3775 deploy/undeploy unittest zimlet

### PR DESCRIPTION
**Pull request by tmclane**
Stop using the deployed 'com_zimbra_phone' zimlet for SOAP unittests:

    zmsoap -z RunUnitTestsRequest/test=com.zimbra.qa.unittest.TestServiceServlet

    <RunUnitTestsResponse numFailed="0" numSkipped="0" numExecuted="11" xmlns="urn:zimbraAdmin">
      <results>
        <completed name="testUnDeployZimlet" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.41"/>
        <completed name="testUnDeployBadZimletname" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.11"/>
        <completed name="testDeployAdminExtension" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.18"/>
        <completed name="testUnDeployAdminExtension" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.18"/>
        <completed name="testFlushZimlets" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.17"/>
        <completed name="testFlushSkins" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.17"/>
        <completed name="testFlushStrings" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.19"/>
        <completed name="testLoadSkins" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.13"/>
        <completed name="testLoadLocales" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.16"/>
        <completed name="testDeployZimlet" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.14"/>
        <completed name="testDeployBadZimletName" class="com.zimbra.qa.unittest.TestServiceServlet" execSeconds="0.12"/>
      </results>
    </RunUnitTestsResponse>

### CODE REVIEW

File: store/src/java/com/zimbra/qa/unittest/TestServiceServlet.java [hunk 1]
```diff
@@ -223,8 +223,8 @@ public void testDeployZimlet() throws Exception {
         String url = baseURL + "deployzimlet";
         PostMethod method = new PostMethod(url);
         addAuthTokenHeader(method, mbox.getAuthToken().getValue());
-        method.addRequestHeader(ZimletUtil.PARAM_ZIMLET, "com_zimbra_phone");
-        File zimletFile = new File("/opt/zimbra/zimlets/com_zimbra_phone.zip"); //standard zimlet, should always be there. No harm in redeploying it
+        method.addRequestHeader(ZimletUtil.PARAM_ZIMLET, "com_zimbra_unittest");
```
Comment (fatkudu) [inline store/src/java/com/zimbra/qa/unittest/TestServiceServlet.java]: Perhaps use a constant for this zimlet name?

Comment (grishick) [reply store/src/java/com/zimbra/qa/unittest/TestServiceServlet.java]: @fatkudu what do you mean?

Comment (fatkudu) [reply store/src/java/com/zimbra/qa/unittest/TestServiceServlet.java]: There are several references to "com_zimbra_unittest" in this file - suggesting using a constant for that, say "TEST_ZIMLET".  Not a big deal but that is what I mean.

**Comment by tmclane** [comment]
Updated with constants for the ZIMLET and the ZIMLET_PATH

**Pull request merged**